hw2/03-count.js

```diff
@@ -1,3 +1,39 @@
 // Add your code here
+const searchWord = document.getElementById("search-word");
+const content = document.getElementById("content");
+const sentence = document.getElementById("content").innerHTML;
+const regex = new RegExp(searchWord, "gi");
 
-input.addEventListener('keydown', handleKeyDown);
+const findAndHighlight = function findAndHighlight() {
+  if (searchWord.value !== "") {
+    let words = content.innerHTML.split(" ");
+    for (let i = 0; i < words.length; i++) {
```

Comment:
I also used a `for` loop for this logic, however another student pointed out that Airbnb prefers to use array functions like `map, foreach` instead. [Airbnb reference](https://github.com/airbnb/javascript#iterators--nope)

Reply:
Hi Stuart. Thanks for your review. I have fixed this.